ambry-clustermap/src/main/java/com/github/ambry/clustermap/HelixParticipantMetrics.java

```diff
@@ -32,26 +32,27 @@
   // no need to record exact number of "dropped" partition, a counter to track partition-dropped events would suffice
   final Counter partitionDroppedCount;
 
-  HelixParticipantMetrics(MetricRegistry metricRegistry) {
+  HelixParticipantMetrics(MetricRegistry metricRegistry, String zkConnectStr) {
```

Comment:
Minor: my only concern here is that the metric names will look convoluted.  But I don't have a better suggestion either for making them distinct.

Reply:
Perhaps in the common case where there is only one participant, you can skip adding the zkConnectStr to the metrics (assuming the caller knows this).

Reply:
Good idea, will try to skip zkConnectStr if there is only one participant.